### ISSUE TITLE
SELECT nonsense >_>

### ISSUE THREAD

**Issue by XxBloodyHell** [XxBloodyHell/DBDOGMS#1]
One of the major features of the bot is to allow users and admins to query vs the database to see what is in there and be able to keep up to date with where users are and have stored information on a daily/hourly/secondly basis(I know I will be checking every 1 second) but anyways, in order to do this, I need to create a FEW handlers for dynamic SELECT queries('find' argument in this db) to handle all that I Need to return to chat upon query made by (user).

The ONLY thing stopping me from creating 999 of these is my problem below V

  Okay so let's go over the problem.
  The db.find method below outputs the information I need to query and send back to the discord chat only in the function(err,docs).
  When outputting docs using bot.sendMessage(docs); the bot returns [object Object].
  Thanks to the help of Kat and Lido, I've realized somewhat of the direction I have to follow in converting the Object
  into string to be parsed and fed back properly.
  I know the direction but not the method to use, where I do sort of want to figure it out myself, any help / pointer is much welcomed.
  
```
  if(message.content.startsWith("poo")) {
    db.find({_id : message.author.id}, function (err,docs){ /* this gonna make me lose my mind */ })
  };
```

**Comment by lidocaine** [comment]
Fixed it for you; see Pull Request #2

**Comment by XxBloodyHell** [comment]
Resolved by basegod Lidocaine.
Will continue to fix my poop code around his god tier code.
